adapters/Vungle/Public/Headers/VungleAdNetworkExtras.h

```diff
@@ -37,4 +37,6 @@
 
 @property(nonatomic, copy) NSString *_Nullable playingPlacement;
 
+@property(nonatomic, copy) NSNumber *_Nullable orientations;
```

Comment:
How about let publishers pass a `UIInterfaceOrientationMask` here? I assume Vungle's SDK does the massaging of this parameter. API doc can say only `UIInterfaceOrientationLandscape`, `Portrait`, or `All` are supported.

Reply:
@joshuajplagonera  Hi joshuajplagonera, because Vungle SDK also uses NSNumber type for orientations options, here we NSNumber type is more convenient. Thanks a lot. 😊